application/models/Stations.php

```diff
@@ -496,6 +498,19 @@ public function check_station_against_callsign($stationid, $callsign) {
 		}
 		return false;
 	}
+
+	// Function return db value from station_profil field //
+	public function get_station_info($id,$field=null,$default_return=false) {
```

Comment:
whole function is redundant to "profile". I see you implemented a "whitelist" so only eqsl_defaultqslmsg is allowed. But please use "profile"